account_invoice_label_certisys/__openerp__.py

```diff
@@ -19,15 +19,17 @@
 ##############################################################################
```

Comment:
could you adapt it to conform with the new copyright mention. Check out it [here](https://github.com/coopiteasy/addons/blob/9.0/hr_timesheet_archived_analytic/__openerp__.py#L2). add my name and yours